caikit/runtime/http_server.py

```diff
@@ -451,7 +451,7 @@ def _get_pydantic_type(cls, field_type: type) -> type:
         if get_origin(field_type) is Annotated:
             return cls._get_pydantic_type(get_args(field_type)[0])
         if get_origin(field_type) is Union:
-            return Union.__getitem__(
+            return Union.__getitem__(  # pylint: disable=unnecessary-dunder-call
```

Comment:
This only failed on certain python versions, but I figured it was worth throwing in there

Reply:
So, it _does_ work if you write it that way, but my inline linter in VS Code goes crazy for it (lots of red squiggles) because `Union[...]` is supposed to have more than one arg and the args are supposed to be types. The fact that `Union` has special semantics for `tuple` appears to not be handled. If it's just me, we can switch it and I can try to find out how to disable this in whatever linter system VS code auto-installed for me.

Reply:
Oh super interesting. Does star-expansion fix your lint issue? Both of these seem to work and neither gets a complaint from Pycharm:
```
some_types = (int, str, bool)
thing = Union[some_types]
thing2 = Union[*some_types]

thing == thing2 # True
thing is thing2 # True
```